mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+hosts
```

Comment:
Is there a reason for the accessors?

Reply:
no you right ,  we should let this choice to the user

Reply:
I will remove it